.circleci/config.yml

```diff
@@ -359,11 +359,28 @@ commands:
       - run:
           name: Install cargo deny, about, edit
           command: |
+            if [[ ! -f "$HOME/.cargo/bin/cargo-binstall$EXECUTABLE_SUFFIX" ]]; then
+              cargo install --locked --version 1.6.4 cargo-binstall
```

Comment:
I'm unsure about using cargo-binstall here. This adds a third party source of executables that is less scrutinized the other install sources we have (apt and crates.io)

Reply:
I think I'll test https://github.com/MaulingMonkey/cargo-local-install which would allow us to add the executables to Cargo.toml where it's more visible than deeply embedded, and this way we could make sure that xtask can rely directly on it

Reply:
Super Fair. I'm also happy to remove this if it's the contentious part of the PR, but would rather land the rest of the PR.  We could also try `--only-signed`, but not sure if that's even trusted enough yet?

Reply:
I've reverted `binstall` in e623cce.